Controllers/AddInterventionViewController.swift

```diff
@@ -603,7 +603,10 @@ class AddInterventionViewController: UIViewController, UITableViewDelegate, UITa
 
   // MARK: - Core Data
 
-  func createIntervention() {
+  @IBAction func createIntervention() {
+    if !checkErrorsInFunctionOfIntervention() {
```

Comment:
```suggestion
    if !checkErrorsAccordingInterventionType() {
```

I think "in function of" doesn't really mean what we want